schemes/hack_shallow/cmfmca.meta

```diff
@@ -0,0 +1,281 @@
+[ccpp-table-properties]
+  name = cmfmca
+  type = scheme
```

Comment:
i believe this should include the wv_saturation dependency

dependencies = ../../to_be_ccppized/wv_saturation.F90

Reply:
Added, thank you!